src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -198,7 +198,8 @@ kibana_vars=(
     xpack.alerting.healthCheck.interval
     xpack.alerting.invalidateApiKeysTask.interval
     xpack.alerting.invalidateApiKeysTask.removalDelay
-    xpack.alerting.defaultRuleTaskTimeout
```

Comment:
Not sure if this should be removed since it's still valid for 7.16 - 8.1. @jbudz Are the docker allowlist configs versioned?

Reply:
Not on the kibana side.  If the config can be set in any form (deprecated, etc) _now_ it should remain in the allow list.

Reply:
Thanks @jbudz 

@ersin-erdal sounds like this should stay in the docker allowlist.